src/components/20-molecules/policy-features/README.md

```diff
@@ -0,0 +1,109 @@
+# Policy features
+
+With this component you can highlight as much `features` as you want. You can set a main title and put `features` items as childs to it. This component changes sizes according to the viewport.
+
+## Usage
+
+**Important:** If this component needs to run in Internet Explorer 11, [you need to use our polyfill](https://github.com/axa-ch/patterns-library/tree/develop/src/components/05-utils/polyfill).
+
+```bash
+npm install @axa-ch/policy-features
+```
+
+```js
+import '@axa-ch/policy-features';
+```
+
+```html
+<axa-policy-features title="Your main title" axaStyle="dark-indigo">
+  <axa-policy-features-item
+    title="The title of item"
+    icon="email"
+    description="A text which can be as long as you want."
+  ></axa-policy-features-item>
+</axa-policy-features>
+```
+
+### React
+
+You can use this component directly in your JSX without further React-ification (see Usage above).
+
+### Pure HTML pages
+
+Import the policy-features-defining script and use a policy-features like this:
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Your awesome title</title>
+  </head>
+  <body>
+    <axa-policy-features title="Your main title" axaStyle="dark-indigo">
+      <axa-policy-features-item
+        title="The title of item"
+        icon="email"
+        description="A text which can be as long as you want."
+      ></axa-policy-features-item>
+    </axa-policy-features>
+    <script src="node_modules/@axa-ch/policy-features/dist/index.js"></script>
+  </body>
+</html>
+```
+
+## Properties
+
+### Variant
+
+| Attribute                | Details                                                          |
+| ------------------------ | ---------------------------------------------------------------- |
+| `axaStyle="default"`     | background is `dark-indigo` and font color is set automatically. |
+| `axaStyle="dark-indigo"` | background is `dark-indigo` and font color is set automatically. |
+| `axaStyle="axa-blue"`    | background is `axa-blue` and font color is set automatically.    |
+| `axaStyle="wild-sand"`   | background is `wild-sand` and font color is set automatically.   |
+| `axaStyle="white"`       | background is `white` and font color is set automatically.       |
+
+### title
+
+The attribute `title` (optional) specifies the main title at the top.
+
+### axaStyle
+
+The attribute `axaStyle` (optional) specifies the background and the font color of this component and all its children.
+
```

Comment:
Unnecessary linebreak. If you use prettier, this whole readme would look way better without any efforts from you. It also automatically improves the tables. :)